### ISSUE TITLE
Invalid data-type for array in EarthLocation.of_address

### ISSUE THREAD

**Issue by Happyigr** [arvindb95/Nakshatra_calculator#2]
Hi, i tried to use your code, but it is not working, i read the docs, but that havent help. I have this error

![image](https://github.com/arvindb95/Nakshatra_calculator/assets/78278275/e34a1c4f-c0bb-45c9-889e-1cab2e64425b)

What should i do?

**Comment by arvindb95** [comment]
Hi. Thanks for reaching out. Can you share what your inputs (especially,
the location string) were to the function? I think that is where things are
going wrong.

Regards
Arvind

**Comment by Happyigr** [comment]
hi, thank you for your answer, i have done it with the initial location:
`location = "Ayodhya, India"`

**Comment by Happyigr** [comment]
Maybe, because i am on linux, smth goes wrong?

**Comment by arvindb95** [comment]
No. Actually, even I use Linux. So it should work. I had tested the code on
my Ubuntu 22.04 machine. I should test it again and then get back to you.
Maybe this is an astropy issue. A newer version of astropy may have
breaking changes. I'll check.

Regards
Arvind

**Comment by arvindb95** [comment]
I'll ping you in the afternoon tomorrow (about 12 hours from now) after
testing it. By the way, both inputs that you are giving are the same
default ones right?

**Comment by Happyigr** [comment]
Thanks, i will wait. I tried it also with the 6.0 astropy version and newer version of numpy, but nothing works for me(
Ya, that were the inputs, that you had written in your code :)
And the input from examples of astropy

**Comment by Happyigr** [comment]
Also when i tried to do it in python (whitout your code) that havent work for me.

**Comment by Happyigr** [comment]
How do you think, can we solve this problem?)

**Comment by arvindb95** [comment]
Hi,

Extremely sorry for the lack of response. I have been travelling a lot and I got bogged down by other things. I was just able to get back to this. 

I see that you have the input `"1002 Holy Grail Court, St. Louis, MO"` in line 93 of `nakshatra_calculations.py`. 

Did you modify that line?  It should read `observing_location = EarthLocation.of_address(location)`. 

The input in the end of the code before calling the `calc_nakshatra_tithi` function should be `location = "1002 Holy Grail Court, St. Louis, MO"` instead of `location = "Ayodhya, India"` for your case. 

Let me know if this corrects the problem that you are having.

Regards
Arvind

**Comment by Happyigr** [comment]
Hi, i tried it, when i cloned the repo, and donesn't change anything, but this doesn't work for me :(
And i tried your suggestion and the error doesn't goes away((

Maybe a python version can be a problem? Because i think, that you have wrote it on python 2 and now is the python 3?

**Comment by arvindb95** [comment]
Hello,

My code is written in Python 3 only. Can you send me the inputs that you
are using?

Or the full error message?

Regards
Arvind

**Comment by Happyigr** [comment]
These are my inputs: 
location = "1002 Holy Grail Court, St. Louis, MO"


And this is the error mssg
![image](https://github.com/arvindb95/Nakshatra_calculator/assets/78278275/83a437c4-6b2f-430a-a858-348e1341515e)

**Comment by arvindb95** [comment]
Hello,

From your error message, `astropy` seems to be the problem. Can you try the following script in a separate code :
```
from astropy.coordinates import EarthLocation

location = "1002 Holy Grail Court, St. Louis, MO"

observing_location = EarthLocation.of_address(location)

print(observing_location)
```
If this doesn't work, I suggest reinstalling `astropy`. 

Regards
Arvind

**Comment by Happyigr** [comment]
i have reinstalled that, but that haven't help me also.
I tried to run this code in python command line, but this gives the same error

**Comment by arvindb95** [comment]
Hi,

What is the astropy version that you are using?

Would it be possible for you to create a virtual environment and do a fresh
install of all the required packages? That way, we can check.

Regards
Arvind

**Comment by Happyigr** [comment]
I have always created a new virtual environment for this project, and installed all from requremnets.txt from your repo.
All the versions are equal to those in the txt file)

**Comment by arvindb95** [comment]
Hi,

I am not sure what is exactly wrong. I tested the code in my virtual environment (with the same packages) and with your input for location, and it ran fine. Shoot me an email at arvind6895@gmail.com and we can try to fix this from there.

Regards
Arvind
